packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -239,11 +263,17 @@ var getDateTime = function (dateTimeStringRaw) {
   }
   var dateTimeString = dateTimeStringRaw?.trim();
 
+  if (incompleteIso8601(dateTimeString)) {
+    return correctIsoFormat(dateTimeString);
+  }
+
   if (alreadyValidDateTime(dateTimeString)) {
     return dateTimeString;
   }
 
-  if (!validDatetimeString(dateTimeString)) return "";
+  if (!validDatetimeString(dateTimeString)) {
```

Comment:
whats the difference between the
`alreadyValidDateTime` and the 
`validDatetimeString` function. Do we need both?

Reply:
I didn't really touch this part of the code besides wrapping the return with the curly braces.

But the difference seems to be that the first one provides a fairly strict format that's close to ISO and causes a return if the check is passed.

The next function does a rougher check if the datetime is following a more general date format. If that check is passed, we then proceed to take it apart and build a date with an expected format